### DISCUSSION TITLE
using forge init throws an error

### DISCUSSION THREAD

**Discussion by Love4codes** [foundry-rs/foundry#4076] [Q&A] [answered]
hello, please i have been trying to use foundry, but whenever i use forge init it throws an error. please how do i resolve this issue?

![Screenshot 2023-01-12 at 12 52 44](https://user-images.githubusercontent.com/103859872/212060270-3558e9b5-43ef-4307-84f6-d6a0dd3ee02e.png)

**Comment by ghost** [comment]
i need the same error

**Comment by ghost** [comment]
i know the reason,because the folder is empty.https://book.getfoundry.sh/projects/working-on-an-existing-project

**Comment by ghost** [comment, marked as answer]
I success，the reason is git.you can try run 'git status' first,then run 'git add -u',and mkdir new empty folder,then run

$ forge init --template https://github.com/foundry-rs/forge-template hello_template

then,you can see the forge lib-std and sol will here.

**Reply by Love4codes** [reply]
thanks
